pkgs/development/python-modules/pywbem/default.nix

```diff
@@ -1,50 +1,57 @@
-{ stdenv, buildPythonPackage, fetchFromGitHub, libxml2
-, m2crypto, ply, pyyaml, six
-, httpretty, lxml, mock, pytest, requests
+{ lib, buildPythonPackage, fetchPypi, libxml2
+, m2crypto, ply, pyyaml, six, pbr, pythonOlder, isPy37, python
+, httpretty, lxml, mock, pytest, requests, decorator, unittest2
 }:
 
 buildPythonPackage rec {
   pname = "pywbem";
-  version = "0.10.0";
+  version = "0.12.6";
 
-  src = fetchFromGitHub {
-    owner  = "pywbem";
-    repo   = "pywbem";
-    rev    = "v${version}";
-    sha256 = "0jcwklip03xcni0dvsk9va8ilqz21g4fxwqd5kzvv91slaadfcym";
+  # Support added in master https://github.com/pywbem/pywbem/commit/b2f2f1a151a30355bbc6652dca69a7b30bfe941e awaiting release
+  disabled = isPy37;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1dc6b745rrys600n05apdf6lb2vv5arlcwv7aiz9whgkbcd9qhki";
   };
 
-  patches = [
-    # fix timezone handling so the tests pass again. Can go when 0.10.1 is released
-    # https://github.com/pywbem/pywbem/issues/755#issuecomment-327508681
-    ./make_cimdatetime_timezone_aware.patch
+  propagatedBuildInputs = [
+    mock
+    pbr
+    ply
+    pyyaml
+    six
+  ] ++ lib.optionals (pythonOlder "3.0") [ m2crypto ];
+
+  checkInputs = [
+    decorator
+    httpretty
+    libxml2
+    lxml
+    pytest
+    requests
+    unittest2
   ];
 
-  propagatedBuildInputs = [ m2crypto ply pyyaml six ];
+  postPatch = ''
+    # Uses deprecated library yamlordereddictloader
+    rm testsuite/test_client.py
 
-  checkInputs = [ httpretty lxml mock pytest requests ];
+    # Wants `wbemcli` in PATH
+    rm testsuite/test_wbemcli.py
```

Comment:
I feel like there's definitely a way to get this test working.
Suggestions are desired.

Reply:
maybe using `installCheckPhase` helps. Also I am not 100% sure, this will add `wbemcli` to PATH. However PATH=$PATH:$out/bin should work then.